src/main/steps/applicant1/check-your-answers/get.ts

```diff
@@ -0,0 +1,16 @@
+import autobind from 'autobind-decorator';
+import { Response } from 'express';
+
+import { Checkbox } from '../../../app/case/case';
+import { AppRequest } from '../../../app/controller/AppRequest';
+import { GetController } from '../../../app/controller/GetController';
+
+@autobind
+export default class CheckYourAnswersGetController extends GetController {
+  public async get(req: AppRequest, res: Response): Promise<void> {
+    req.session.userCase.applicant1IBelieveApplicationIsTrue = Checkbox.Unchecked;
```

Comment:
this is abit tricky, even tho you unchecked it here, itll still be checked in the casedata.

Reply:
I'm okay with it not being reset in the case data. If that becomes a problem we can think of a better way